src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -1846,8 +1842,7 @@ public static string CutToClipboard(TextField textField)
         /// Paste the most recently copied/cut text from the clipboard and into the text control.
```

Comment:
Please describe how is it pasted. Such details can go into \<remarks\> tag.
What I'm interested in: does the pasted text replace the all the texteditor contents, or is it added at the end, replaces the selected text (that's what I'm used to in text editors on PC)?

Reply:
@wiertel , 
Done.
Please could check and let me know what you see.